xpcs_online_boost_client.py

```diff
@@ -27,11 +27,11 @@ def arg_parse():
     parser = argparse.ArgumentParser()
     parser.add_argument('--experiment', help='Name of the DM experiment', default='test-xpcs-local-workflow-2023.12.19-01')
     parser.add_argument('--hdf', help='Path to the hdf (metadata) file',
-                        default='A001_Aerogel_1mm_att6_Lq0_001_0001-1000.hdf')
-    parser.add_argument('-r', '--raw', help='Path to the raw data file. Multiple formats (.imm, .bin, etc) supported',
-                        default='A001_Aerogel_1mm_att6_Lq0_001_00001-01000.imm')
-    parser.add_argument('-q', '--qmap', help='Path to the qmap file',
-                        default='comm201901_qmap_aerogel_Lq0.h5')
+                        default='/gdata/dm/8IDI/2024-1/zhang202402_2/data/H001_27445_QZ_XPCS_test-01000/H001_27445_QZ_XPCS_test-01000.hdf')
+    parser.add_argument('--raw', help='Path to the raw data file. Multiple formats (.imm, .bin, etc) supported',
+                        default='/gdata/dm/8IDI/2024-1/zhang202402_2/data/H001_27445_QZ_XPCS_test-01000/H001_27445_QZ_XPCS_test-01000.h5')
+    parser.add_argument('--qmap', help='Path to the qmap file',
+                        default='/gdata/dm/8IDI/2024-1/zhang202402_2/data/standard_qmaps/eiger4M_qmap_d36_s360.h5')
```

Comment:
I had to add the full path of these files to have them be picked up and run. Using basenames didn't work for me.

Reply:
You're right that is a bit ambiguous. The part that is nice is having some default test datasets we can use to check the flow is operating correctly, as a simple sanity/integration test if any other part of the flow changes.

Right now I don't think we have a way to check if we are using the correct qmap paring. I would consider it "user error" if someone makes that mistake. It would be a really nice addition to prevent someone using an incompatible qmap, or at the very least fail flows with a clear error message. I don't think we have a way to do this at the flows level, I would like to check in with Miaoqi to see if there's a way boost_corr could detect improper qmaps and spit out error messages.